meta-xt-domu-gen3/recipes-tools/images/core-image-weston.bbappend

```diff
@@ -1,6 +1,6 @@
 IMAGE_INSTALL:append = " \
     pciutils \
-    kmscube \
+    ${@bb.utils.contains('MACHINE_FEATURES', 'gsx', 'kmscube', '', d)} \
```

Comment:
Can this be done with overrides syntax?

Something like `IMAGE_INSTALL:gsx:append = kmscube`?

Reply:
We can't use override syntax here. Because
> The default value of OVERRIDES includes the values of the CLASSOVERRIDE, MACHINEOVERRIDES, and DISTROOVERRIDES variables. Another important override included by default is pn-${PN}.

So MACHINE_FEATURES is not part of OVERRIDES.